From 51c6862508e638f87d19513d1c37ef72019fb799 Mon Sep 17 00:00:00 2001
From: Alexander Mikhalitsyn <aleksandr.mikhalitsyn@canonical.com>
Date: Thu, 7 Dec 2023 14:09:59 +0100
Subject: [PATCH 1/3] explicitly define ____cacheline_aligned

See also 37fb3e431 ("OpenZFS 8484 - Implement aggregate sum and use for arc counter")

Signed-off-by: Alexander Mikhalitsyn <aleksandr.mikhalitsyn@canonical.com>
---
 include/sys/zfs_context.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/sys/zfs_context.h b/include/sys/zfs_context.h
index 68c58f955..99aaed9d4 100644
--- a/include/sys/zfs_context.h
+++ b/include/sys/zfs_context.h
@@ -785,5 +785,7 @@ extern fstrans_cookie_t spl_fstrans_mark(void);
 extern void spl_fstrans_unmark(fstrans_cookie_t);
 extern int __spl_pf_fstrans_check(void);
 
+#define	____cacheline_aligned
+
 #endif /* _KERNEL */
 #endif	/* _SYS_ZFS_CONTEXT_H */
-- 
2.34.1

